packages/core/src/domain/configuration.ts

```diff
@@ -250,9 +250,13 @@ function getEndpoint(
 function getHost(intakeType: IntakeType, endpointType: EndpointType, site: string) {
   const middleDomain = INTAKE_MIDDLE_DOMAINS[intakeType]
   const endpoint = ENDPOINTS[intakeType][endpointType]
-  const domainParts = site.split('.')
-  const ext = domainParts.pop()
-  return `${endpoint}${middleDomain}${domainParts.join('-')}.${ext}`
+  let suffix = site
+  if (intakeType === 'alternate') {
+    const domainParts = site.split('.')
+    const ext = domainParts.pop()
+    suffix = `${domainParts.join('-')}.${ext}`
+  }
+  return `${endpoint}${middleDomain}${suffix}`
```

Comment:
IMO, making an abstraction for middle domain seems unnecessary and implementation could be clearer without it:

```ts
function getHost(intakeType: IntakeType, endpointType: EndpointType, site: string) {
  const endpoint = ENDPOINTS[intakeType][endpointType]
  if (intakeType === 'classic') {
    return `${endpoint}-http-intake.logs.${site}`
  }
  const domainParts = site.split('.')
  const extension = domainParts.pop()
  const suffix = `${domainParts.join('-')}.${extension}`
  return `${endpoint}.browser-intake-${suffix}`
}
```

Reply:
Now that I see it, I agree.